cloudtools/awacs
# Update services

- Update services to latest IAM policies
- Add Action helper to each service
- Update generator tool for new ARN and input workarounds
- Make sure filenames don't use illegal "-"
- Remove renamed zocalo and deprecated whispersync
